src/Fixtures/AbstractFixture.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\FixtureBundle\Fixtures;
+
+use Doctrine\Bundle\FixturesBundle\ORMFixtureInterface;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Neusta\Pimcore\FixtureBundle\ReferenceRepository\DataObjectReferenceRepository;
+use Symfony\Contracts\Service\Attribute\Required;
+
+abstract class AbstractFixture implements FixtureInterface, ORMFixtureInterface
```

Comment:
Technically, we don't need the `FixtureInterface` because `ORMFixtureInterface` already extends it.